helpers/encodeCallData.ts

```diff
@@ -0,0 +1,62 @@
+import BigNumber from "bignumber.js";
+import { numberToPaddedHex } from "@multiversx/sdk-core/out/utils.codec";
```

Comment:
maybe import directly from `@multiversx/sdk-core/utils.codec`, not from the `/out` dir.

Reply:
TS2307: Cannot find module  @multiversx/sdk-core/utils.codec  or its corresponding type declarations.

Reply:
Interesting. We'll keep that in mind.

Reply:
Also, feel free to:

```
export function numberToPaddedHex(value: bigint | number | BigNumber.Value) {
    let hexableNumber: { toString(radix?: number): string };

    if (typeof value === "bigint" || typeof value === "number") {
        hexableNumber = value;
    } else {
        hexableNumber = new BigNumber(value);
    }

    const hex = hexableNumber.toString(16);
    return zeroPadStringIfOddLength(hex);
}

export function zeroPadStringIfOddLength(input: string): string {
    input = input || "";

    if (input.length % 2 == 1) {
        return "0" + input;
    }

    return input;
}
```